docs/src/main/asciidoc/quickstart.adoc

```diff
@@ -0,0 +1,223 @@
+This quick start walks through using Spring Cloud Zookeeper for Service Discovery and Distributed Configuration.
+
+First, run Zookeeper on your machine, then you can access it and use as Service Registry and Configuration source with Spring Cloud Zookeeper.
```

Comment:
Change to:

First, run Zookeeper on your machine. Then you can access it and use it as a Service Registry and Configuration source with Spring Cloud Zookeeper.